paasta_tools/instance/kubernetes.py

```diff
@@ -81,9 +81,20 @@ def autoscaling_status(
     namespace: str,
 ):
     status = {}
-    hpa = kube_client.autoscaling.read_namespaced_horizontal_pod_autoscaler(
-        name=job_config.get_sanitised_deployment_name(), namespace=namespace
-    )
+    try:
+        hpa = kube_client.autoscaling.read_namespaced_horizontal_pod_autoscaler(
+            name=job_config.get_sanitised_deployment_name(), namespace=namespace
+        )
+    except ApiException as e:
+        if e.status == 404:
```

Comment:
@EvanKrall Can you take a look at this section?  I think this is a better solution to the `paasta status` error than just adding something to `error_message`, because `error_message` prevents the rest of the status output from being displayed.

Reply:
Is there some easy way we can have `status` output a combination of `error_message` and actual responses?

Otherwise, yeah, ship. Anything that makes `paasta status` degrade more gracefully in the face of its many failure modes is good.

Reply:
I think what I'd ideally do here is "add a command-line option that displays errors at the bottom", but that sounds like more work than what I want to do in this PR.  I'll make a ticket, though.